swift/Sources/FlatBuffers/FlatBufferBuilder.swift

```diff
@@ -907,7 +911,11 @@ extension FlatBufferBuilder: CustomDebugStringConvertible {
     /// - Returns: a FieldLoc at index
     @inline(__always)
     func load(at index: Int) -> FieldLoc {
+#if swift(<5.7)
       memory.load(fromByteOffset: index, as: FieldLoc.self)
+#else
+      memory.loadUnaligned(fromByteOffset: index, as: FieldLoc.self)
+#endif
```

Comment:
nit: same here

Reply:
Sure, if it's not needed there is not reason to have it here. Removed.